com.unity.netcode.gameobjects/Runtime/Utility/Ref.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Unity.Multiplayer.Netcode
+{
+    public struct Ref<T> where T: unmanaged
```

Comment:
This entire struct is just the & operator unless I misunderstand something. There's not need for the pinning and fixing here and instead of IsSet we can just check the pointer for null?

Reply:
It's a compile error if I try to take the pointer directly without the fixed() expression, unfortunately. That's because even value types can be members of managed types and there's no way to add a constraint that they must be on stack memory. Ref<> is just... easier to read and avoids having to repeat the fixed() expression every time this pattern emerges.

Reply:
Also, Ref<> contains the unsafe code so the structs using it don't have to declare all their methods to be unsafe.